examples/displayio_layout_cartesian_advanced_test.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: 2021 Jose David M.
+#
+# SPDX-License-Identifier: MIT
+#############################
+"""
+This is a more advance demonstration of a Cartesian widget and some configurable
+parameters.
+"""
+
+import board
+import displayio
+import terminalio
+from adafruit_displayio_layout.widgets.cartesian import Cartesian
+
+# Fonts used for the Dial tick labels
+tick_font = terminalio.FONT
+
+display = board.DISPLAY  # create the display on the PyPortal or Clue (for example)
+# otherwise change this to setup the display
+# for display chip driver and pinout you have (e.g. ILI9341)
+
+
+# Create different Cartesian widgets
+my_group = displayio.Group(max_size=10)
+
+car = Cartesian(
+    x=25,
+    y=10,
+    width=100,
+    height=100,
+    subticks=True,
+)
+my_group.append(car)
+
+car3 = Cartesian(
+    x=150,
+    y=10,
+    width=150,
+    height=100,
+    xrange=(0, 160),
+    axes_stroke=1,
+    axes_color=0x990099,
+    subticks=True,
+)
+my_group.append(car3)
+
+car4 = Cartesian(
+    x=30,
+    y=140,
+    width=80,
+    height=80,
+    axes_stroke=1,
+    tick_color=0xFFFFFF,
+    subticks=True,
+)
+car4.set_widget_style("LightGreen6")
```

Comment:
`set_widget_style` got removed from the Cartesian object, needs to be removed from the example here as well.

Reply:
@FoamyGuy thanks for reviewing. I must forgot that I have that in there, my bad, Thanks for keeping it up the good work :). I remove the reference in https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout/pull/22/commits/001b4efb57a4e10c898a1a8bf1403342c32acc92